### PR TITLE
Docs: Update Readme to the Requirements Reference

### PR DESCRIPTION
**Pull request by ankyit**
Update Readme to the Requirements Reference

/kind documentation

> /kind enhancement
> /kind good-first-issue
> /kind feature
> /kind question
> /kind design
> /sig ai
> /sig iot
> /sig network
> /sig storage
> /sig storage

#### What this PR does / why we need it:
Readme was referencing the wrong link for the documentation. Updated link to the correct one. 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by ankyit** [comment]
Fixes two document issues for the links:
1. Updates the URL for openyurt - resolves typo 
2. Updates Requirement link

**Comment by rambohe-ch** [comment]
@ankyit Thank you for posting pull request. and how about update [README.zh.md](https://github.com/openyurtio/openyurt/blob/master/README.zh.md) file as well?

**Comment by ankyit** [comment]
> @ankyit Thank you for posting pull request. and how about update [README.zh.md](https://github.com/openyurtio/openyurt/blob/master/README.zh.md) file as well?

Sure I will add them and update the pull request.

**Comment by ankyit** [comment]
> > @ankyit Thank you for posting pull request. and how about update [README.zh.md](https://github.com/openyurtio/openyurt/blob/master/README.zh.md) file as well?
> 
> Sure I will add them and update the pull request.

@rambohe-ch Changes for the ZH page applied. Please validate

File: README.zh.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 | 最新发布：2022-05-27  OpenYurt v0.7.0 请查看 [CHANGELOG](CHANGELOG.md) 来获得更多更新细节.|
 | 第一个发布：2020-05-29 OpenYurt v0.1.0-beta.1 |
 
-OpenYurt (官网: <https://openyurt.io>) 是基于Upstream Kubernetes构建的，现在是托管在云原生基金会(CNCF) 下的 [沙箱项目](https://www.cncf.io/sandbox-projects/).
+OpenYurt (官网: <https://openyurt.io) 是基于Upstream Kubernetes构建的，现在是托管在云原生基金会(CNCF) 下的 [沙箱项目](https://www.cncf.io/sandbox-projects/).
```
Comment (Congrool) [inline README.zh.md:20]: I think `<` should also be removed.

File: README.zh.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ OpenYurt 遵循经典的云边一体化架构。
 
 ## 开始之前
 
-安装OpenYurt前，请检查[资源和系统要求](./docs/resource-and-system-requirements-cn.md)
+安装OpenYurt前，请检查[资源和系统要求]([./docs/resource-and-system-requirements-cn.md](https://github.com/openyurtio/openyurt.io/blob/master/i18n/zh/docusaurus-plugin-content-docs/current/usage-conditions/resource-and-system-requirements.md))
```
Comment (Congrool) [inline README.zh.md:62]: It cannot jump to the link site. 
It should be `[资源和系统要求](https://github.com/openyurtio/openyurt.io/blob/master/i18n/zh/docusaurus-plugin-content-docs/current/usage-conditions/resource-and-system-requirements.md)`

**Comment by rambohe-ch** [comment]
@ankyit please fix the comments that come from @Congrool

**Comment by ankyit** [comment]
@rambohe-ch , @Congrool  Both updates were applied, Please validate.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**